languages/ruby/exercises/concept/ostruct/.docs/introduction.md

```diff
@@ -0,0 +1,33 @@
+Ruby comes with a Standard Library (often shortened to "stdlib") - a collection of classes for working with things such as dates, json, and networking.
+It also provide some useful functionality for making your code easier to work with.
+
+`OpenStruct` is part of the Standard Library and allows you to easily create an object from a hash. Rather than having to access using hash keys, `OpenStruct` instead allows us to use methods to access and set values.
```

Comment:
```suggestion
`OpenStruct` is part of the Standard Library and allows you to easily create an object from a `Hash`. Rather than having to access using `Hash` keys, `OpenStruct` instead allows us to use methods to access and set values.
```